package.json

```diff
@@ -106,5 +108,8 @@
     "not dead",
     "not op_mini all"
   ],
-  "packageManager": "yarn@3.1.1"
+  "packageManager": "yarn@3.1.1",
+  "resolutions": {
+    "react-scripts": "patch:react-scripts@npm:5.0.0#.yarn/patches/react-scripts-npm-5.0.0-14ea4c40dd.patch"
```

Comment:
What is this patch about? polyfills? I thinks if the benefit is only build speed then it's better to stick with the vanilla setup without patches that needs to be maintained.

Reply:
This patch does 2 things:

1. Allows react-scripts 5 to work with this codebase (the dependency `sjcl` requires these polyfills, and won't compile without them)
2. Improves build speed (this part can be reverted, as it is optional)

So without refactoring out `sjcl`, react-scripts either has to stay on 4 or be patched.